terraform/shared/shared.tf.json.template.py

```diff
@@ -41,7 +43,9 @@ def conformance_pack(name: str) -> str:
     # https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cis-controls.html#securityhub-cis-controls-3.1
     CloudTrailAlarm(name='api_unauthorized',
                     statistic='Average',
-                    filter_pattern='{($.errorCode="*UnauthorizedOperation") || ($.errorCode="AccessDenied*")}'),
+                    filter_pattern='{($.errorCode="*UnauthorizedOperation") || ($.errorCode="AccessDenied*")}',
+                    threshold=12,
```

Comment:
Consider 12, since we do get ~11 undesired (AWS Config and SSM) alarms /day…

![Screenshot 2024-05-03 at 2 30 21 PM](https://github.com/DataBiosphere/azul/assets/19215510/91d6063c-4e7d-4c09-b35f-567f15bfd82e)

… and the alarm's [`comparison_operator` is `GreaterThanOrEqualToThreshold`](https://github.com/DataBiosphere/azul/blob/0065686fb3720437a5c958645597e806682b0352/terraform/shared/shared.tf.json.template.py#L522C45-L522C74).